user-guide/live-traffic/live-traffic.md

```diff
@@ -59,7 +59,7 @@ Use the **Advanced Filters** to apply filters based on the Request and Response
 
 Currently, the **Advanced Filters** entries are using **AND** logical operator. Each value is complying with the **EXACT** match.
```

Comment:
...are using the **AND** logical operator. - has to be defined acc to grammar rules.